lib/pbench/server/api/__init__.py

```diff
@@ -96,6 +99,7 @@ def create_app():
     app.config["BIND_HOST"] = app.config_server.get("bind_host")
     app.config["WORKERS"] = app.config_server.get("workers")
 
+    _MAX_UPLOAD_SIZE = filesize_bytes(app.config["MAX_CONTENT_LENGTH"])
```

Comment:
Why not just set `app.config["MAX_CONTENT_LENGTH"]` so the server can get the converted value through the normal config dict mechanism rather than introducing a global? Just combine the conversion with the access in line 96 up above. There's no reason to be storing the original text.

Reply:
We use the "humanized" value of MAX_CONTENT_LENGTH on line #324 in the abort message. We will have to rehumanize the bytes value for the abort message thats why decided to use the constant variable. However, if you think its better to drop global variable I can do that

Reply:
You could split it into two keys, though, and still keep it in the dict.

```
app.config["MAX_CONTENT_LENGTH_INPUT"] = app.config_server.get("rest_content_max_length")
app.config["MAX_CONTENT_LENGTH"] = filesize_bytes(app.config["MAX_CONTENT_LENGTH_INPUT"])
```
However, there's also the question of whether we want to "optimally humanize" the value we use or just spit back the config file value they typed. E.g., is "1024 KB" reported as "1024 KB" or as "1 GB"??? In the latter case we'd want to re-humanize the binary value anyway.

Reply:
yeah I think we can add config key to hold bytes number. Regarding your second concern, I believe its manageable since the default config file is in our control

Reply:
Pete's fixes for spacing and lack of units could be taken as an argument against the "in our control" reasoning!

But again, I don't object to storing and emitting the original text; I just don't like the global. However, since we're only emitting the text version on a failure path, I don't see the cost of re-humanizing as an issue, either, and there might be some advantages to "normalizing" the notation.

Reply:
I agree, lesser the globals better it is